fields.go

```diff
@@ -74,6 +74,21 @@ func appendFieldList(dst []byte, kvList []interface{}) []byte {
 			default:
 				dst = enc.AppendInterface(dst, m)
 			}
+
+			if stack && ErrorStackMarshaler != nil {
+				dst = enc.AppendKey(dst, ErrorStackFieldName)
```

Comment:
Given that a `Fields()` can print multiple errors, should we append the key to `ErrorStackFieldName`?
If we do that, we could enable the display of _stacks_ for `[]error` as well.